pkgs/development/python-modules/ledger_bitcoin/default.nix

```
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools
+, ledgercomm
+, packaging
+, typing-extensions
+ }:
+
+buildPythonPackage rec {
+  pname = "ledger_bitcoin";
```

use dash instead of underscore:

```suggestion
  pname = "ledger-bitcoin";
```